app/_includes/docs-sidebar.html

```diff
@@ -1,5 +1,47 @@
 <aside class="docs-sidebar">
   <i class="fa fa-times close-sidebar"></i>
+
+  <div class="sidebar-title-container">
+    <div class="sidebar-title">
+      {% if page.edition == 'enterprise' %}{{site.ee_product_name}}
+      {% elsif page.edition == 'konnect' %}{{site.konnect_saas}}
+      {% elsif page.edition == 'mesh' %}{{site.mesh_product_name}}
+      {% elsif page.edition == 'deck' %}decK
+      {% elsif page.edition == 'kubernetes-ingress-controller' %}{{site.kic_product_name}}
+      {% elsif page.edition == 'gateway-oss' %}{{site.ce_product_name}}
+      {% elsif page.edition == 'getting-started-guide' %}Getting Started with Kong Gateway
+      {% endif %}
```

Comment:
After having myself and a team member test this, we both realized that just having the name of the doc set here without a link is confusing. The flow feels interrupted: if you want to get to another doc set, you have to go to the navbar, to the right, which has historically been a Kong marketing link. While I think we should keep the new nav menu (the nav-v2), we should also turn this into a dropdown similar to versions:
![Screen Shot 2021-03-05 at 3 52 42 PM](https://user-images.githubusercontent.com/54370747/110394527-70d1b600-8021-11eb-9adb-605b6cd654a2.png)